README.md

```diff
@@ -149,9 +149,16 @@ git push -u origin <your_branch_name>
 - *Why clone everything to the machine? Just press "." (dot) after opening the forked repo in a browser tab. See the Magic!🎉*
 - *Want to preview your live changes? Copy and paste the project.html URL into [GitHack](https://raw.githack.com/). You will get 2 URLs for production and development, copy and paste them into a new tab. See the Magic!🎉*
 
+<br>
```

Comment:
You don't have to make changes in README. It's not needed, I closed your issue related to this.

Reply:
Thanks for the suggestion about file hierarchy, will revamp it after GSSoC.